src/optimise1.jl

```diff
@@ -176,10 +186,13 @@ function minimise_icp_constrained(f::Function, x::IntervalBox{N, T}, constraints
             global_minimum = current_minimum
```

Comment:
In general this logic is not correct, since there may be no feasible points in `p`.

Reply:
Yes right..!! this required more work need to be done on this case.